net/pfSense-pkg-WireGuard/files/usr/local/www/wg/vpn_wg_edit.php

```diff
@@ -0,0 +1,687 @@
+			}
+
+			attachhandlers()
+
+		}
+
+		$('.peer_group_' + peernum).find('td').eq(0).text(peernum)
+		$('.peer_group_' + peernum).find('td').eq(1).text($('#pdescr').val())
+		$('.peer_group_' + peernum).find('td').eq(2).text($('#endpoint').val())
+		$('.peer_group_' + peernum).find('td').eq(3).text($('#port').val())
+		$('.peer_group_' + peernum).find('td').eq(4).text($('#ppublickey').val())
+		$('.peer_group_' + peernum).find('td').eq(5).text($('#persistentkeepalive').val())
+		$('.peer_group_' + peernum).find('td').eq(6).text($('#allowedips').val())
+		$('.peer_group_' + peernum).find('td').eq(7).text($('#presharedkey').val())
+		$('.peer_group_' + peernum).find('td').eq(8).text($('#peerwgaddr').val())
+
+		$('#peermodal').modal('hide');
+	});
+
+	// Save the form
+	$('#saveform').click(function () {
+		// For each row in the peers table, construct an array of inputs with the values from the row
+		$('#peertable > tbody').find('tr').each(function (idx) {
+			$('<input>').attr({type: 'hidden',name: 'descp[]', value: $(this).find('td').eq(1).text()}).appendTo(form);
+			$('<input>').attr({type: 'hidden',name: 'endpoint[]', value: $(this).find('td').eq(2).text()}).appendTo(form);
+			$('<input>').attr({type: 'hidden',name: 'port[]', value: $(this).find('td').eq(3).text()}).appendTo(form);
+			$('<input>').attr({type: 'hidden',name: 'publickeyp[]', value: $(this).find('td').eq(4).text()}).appendTo(form);
+			$('<input>').attr({type: 'hidden',name: 'persistentkeepalive[]', value: $(this).find('td').eq(5).text()}).appendTo(form);
+			$('<input>').attr({type: 'hidden',name: 'allowedips[]', value: $(this).find('td').eq(6).text()}).appendTo(form);
+			$('<input>').attr({type: 'hidden',name: 'presharedkey[]', value: $(this).find('td').eq(7).text()}).appendTo(form);
+			$('<input>').attr({type: 'hidden',name: 'peerwgaddr[]', value: $(this).find('td').eq(8).text()}).appendTo(form);
+		});
+
+		$('<input>').attr({type: 'hidden',name: 'save',value: 'save'}).appendTo(form);
+
+		// Recalculate the table hash so the browser doesn't intercept the save
+		tableHash = hashCode($('#peertable').html());
+		$(form).submit();
+	});
+
+	$('#copypubkey').click(function () {
+		$('#publickey').focus();
+		$('#publickey').select();
+		document.execCommand("copy");
+	});
+
+	attachhandlers()
+
+	function attachhandlers() {
+		// Delete a peer
+		$('[id^=killpeer_]').click(function () {
+			var row = this.id.slice('killpeer_'.length)
+			if (confirm('Are you sure you want to delete peer ' + row + '?')) {

```

Comment:
Please reword this to avoid addressing the user directly ("you").

Reply:
This is no longer relevant.